src/main/kotlin/io/appwrite/services/Account.kt

```diff
@@ -0,0 +1,431 @@
+package io.appwrite.services
+
+import io.appwrite.Client
+import io.appwrite.exceptions.AppwriteException
+import okhttp3.Cookie
+import okhttp3.Response
+import java.io.File
+
+class Account(private val client: Client) : BaseService(client) {
+
+    /**
+     * Get Account
+     *
+     * Get currently logged in user data as JSON object.
+     *
+     * @return [Response]     
+     */
+    @JvmOverloads
+    @Throws(AppwriteException::class)
+    suspend fun get(): Response {
+        val path = "/account"
+        val params = mapOf<String, Any?>(
+        )
+
+        val headers = mapOf(
+            "content-type" to "application/json"
+        )
+
+        return client.call("GET", path, headers, params)
+    }
+    
+    /**
+     * Delete Account
+     *
+     * Delete a currently logged in user account. Behind the scene, the user
+     * record is not deleted but permanently blocked from any access. This is done
+     * to avoid deleted accounts being overtaken by new users with the same email
+     * address. Any user-related resources like documents or storage files should
+     * be deleted separately.
+     *
+     * @return [Response]     
+     */
+    @JvmOverloads
+    @Throws(AppwriteException::class)
+    suspend fun delete(): Response {
+        val path = "/account"
+        val params = mapOf<String, Any?>(
+        )
+
+        val headers = mapOf(
+            "content-type" to "application/json"
+        )
+
+        return client.call("DELETE", path, headers, params)
+    }
+    
+    /**
+     * Update Account Email
+     *
+     * Update currently logged in user account email address. After changing user
+     * address, user confirmation status is being reset and a new confirmation
+     * mail is sent. For security measures, user password is required to complete
+     * this request.
+     * This endpoint can also be used to convert an anonymous account to a normal
+     * one, by passing an email address and a new password.
+     *
+     * @param email
+     * @param password
+     * @return [Response]     
+     */
+    @JvmOverloads
+    @Throws(AppwriteException::class)
+    suspend fun updateEmail(
+		email: String,
```

Comment:
Service functions with parameters indentation could be changed to match 4 spaces from function start, parameter list closing line could be changed to align with function start

Reply:
:+1: